thatkitebot/cogs/utilitiescog.py

```diff
@@ -64,6 +65,55 @@ async def status(self, ctx):
             embed.color = ec.purple_violet
         await ctx.send(embed=embed)
 
+    @commands.cooldown(1, 5, commands.BucketType.user)
+    @commands.command(pass_context=True, aliases=["serverinfo", "guildinfo", "server"])
+    async def guild(self, ctx):
+        """
+        Displays information about the server.
+        """
+        guild = ctx.guild
+        embed = discord.Embed()
+        embed.title = guild.name
+        embed.description = guild.description
+        embed.add_field(name="Owner", value=guild.owner.mention)
+        embed.add_field(name="Members", value=guild.member_count)
+        embed.add_field(name="Channels", value=len(guild.channels))
+        embed.add_field(name="Roles", value=len(guild.roles))
+        embed.add_field(name="Created at", value=guild.created_at.strftime("%d/%m/%Y %H:%M:%S"))
+        age = relativedelta(datetime.now(timezone.utc), guild.created_at)
+        embed.add_field(name="Age", value=(f"{(str(age.years) + ' years, ' if age.years != 1 else ' year' + ', ') if age.years != 0 else ''}{(str(age.months) + ' months, ' if age.months != 1 else ' month, ') if age.months != 0 else ''}{age.days} {'days' if age.days != 1 else 'day'}"))
+        if guild.icon:
+            embed.set_thumbnail(url=guild.icon)
+        embed.color = ec.lime_green
+        await ctx.send(embed=embed)
+
+    @commands.cooldown(1, 5, commands.BucketType.user)
+    @commands.command(pass_context=True, aliases=["userinfo", "user"])
+    async def whois(self, ctx, user: discord.Member = None):
+        """
+        Displays information about a user.
+        """
+        if user is None:
+            user = ctx.author
+        embed = discord.Embed()
+        embed.title = user.name
+        embed.description = user.mention
+        embed.add_field(name="ID", value=user.id)
+        embed.add_field(name="Server joined", value=user.joined_at.strftime("%d/%m/%Y %H:%M:%S"))
+        age = relativedelta(datetime.now(timezone.utc), user.joined_at)
+        embed.add_field(name="Server age", value=(f"{(str(age.years) + ' years, ' if age.years != 1 else ' year' + ', ') if age.years != 0 else ''}{(str(age.months) + ' months, ' if age.months != 1 else ' month, ') if age.months != 0 else ''}{age.days} {'days' if age.days != 1 else 'day'}"))
```

Comment:
that is a bit too pythonic even for me :trollface:

Reply:
![image](https://user-images.githubusercontent.com/25513498/201421328-b5bd398d-dbfb-48ad-b6fd-cd5478f396d5.png)